There is an issue with WOW64 build:
```
In file included from /home/runner/work/box64/box64/src/dynarec/arm64/dynarec_arm64_helper.c:11:
/home/runner/work/box64/box64/wine/wow64/../../src/emu/x64emu_private.h:143:5: error: unknown type name 'siginfo_t'; did you mean 'sysinfo_t'?
  143 |     siginfo_t   deferred_siginfo[MAX_SIGNAL+1];
      |     ^~~~~~~~~
      |     sysinfo_t
/home/runner/work/box64/box64/wine/wow64/../../src/include/sysinfo.h:21:3: note: 'sysinfo_t' declared here
   21 | } sysinfo_t;
      |   ^
1 error generated.
```

--- a/src/custommem.c
+++ b/src/custommem.c
@@ -16,7 +16,9 @@
 #include "callback.h"
 #include "x64trace.h"
 #include "custommem.h"
+#ifndef _WIN32
 #include "signals.h"
+#endif
 #include "khash.h"
 #include "threads.h"
 #include "rbtree.h"
@@ -563,6 +565,11 @@ static uintptr_t    defered_prot_p = 0;
 static size_t       defered_prot_sz = 0;
 static uint32_t     defered_prot_prot = 0;
 static mem_flag_t   defered_prot_flags = MEM_ALLOCATED;
+
+#ifdef _WIN32
+void enter_critical_section(void) {}
+void leave_critical_section(void) {}
+#endif
 
 static void setProtection_generic(uintptr_t addr, size_t sz, uint32_t prot, mem_flag_t flags);
 #define LOCK_PROT()               \

--- a/src/emu/x64emu.c
+++ b/src/emu/x64emu.c
@@ -43,11 +43,13 @@ static uint32_t x86emu_parity_tab[8] =
 
 static void reset_deferred_signal(x64emu_t* emu)
 {
+    #ifndef _WIN32
     emu->critical_section = 0;
     emu->deferred_signal_processing = 0;
     emu->deferred_signal_count = 0;
     memset((void*)emu->deferred_signal_pending, 0, sizeof(emu->deferred_signal_pending));
     memset(emu->deferred_siginfo, 0, sizeof(emu->deferred_siginfo));
+    #endif
 }
 
 static void internalX64Setup(x64emu_t* emu, box64context_t *context, uintptr_t start, uintptr_t stack, int stacksize, int ownstack)

--- a/src/emu/x64emu_private.h
+++ b/src/emu/x64emu_private.h
@@ -136,11 +136,13 @@ typedef struct x64emu_s {
     tlsdatasize_t  *tlsdata;
     // other informations
     int         type;       // EMUTYPE_xxx define
+    #ifndef _WIN32
     volatile sig_atomic_t critical_section;
     volatile sig_atomic_t deferred_signal_processing;
     volatile sig_atomic_t deferred_signal_count;
     volatile sig_atomic_t deferred_signal_pending[MAX_SIGNAL+1];
     siginfo_t   deferred_siginfo[MAX_SIGNAL+1];
+    #endif
     #ifdef BOX32
     int         libc_err;   // copy of errno from libc
     int         libc_herr;  // copy of h_errno from libc

